next-packages/serialize/src/index.js

```diff
@@ -75,6 +79,8 @@ if (process.env.NODE_ENV !== 'production') {
   }
 }
 
+let cursor
```

Comment:
a comment explanation why this is kept at module scope would be good for future code readers